frontend/packages/metal3-plugin/src/components/modals/PowerOffHostModal.tsx

```diff
@@ -81,16 +89,83 @@ const isPowerOffSafe = (status: string) => {
   return safeStates.includes(status);
 };
 
+type StatusValidationProps = {
+  status: string;
+  nodePods: FirehoseResult<PodKind[]>;
+  loadError?: any;
+};
+
+const StatusValidations: React.FC<StatusValidationProps> = ({ status, nodePods, loadError }) => {
+  const validations = [];
+
+  if (loadError) {
+    validations.push({
+      title: 'Failed to load data.',
+      description: 'Failed to load subresources.',
+      level: 'danger',
+    });
+  }
+
+  if ([HOST_STATUS_UNKNOWN, ...HOST_HEALTH_ERROR].includes(status)) {
+    validations.push({
+      title: 'The bare metal host is not healthy.',
+      description: 'The host cannot be powered off gracefully untils its health is restored.',
+      level: 'warning',
+    });
+  }
+
+  if (status === NODE_STATUS_STARTING_MAINTENANCE) {
+    validations.push({
+      title: 'The node is starting maintenance.',
+      description:
+        'The node cannot be powered off gracefully until it is either in maintenance or healthy.',
+      level: 'info',
+    });
+  }
+
+  if (status === NODE_STATUS_STARTING_MAINTENANCE) {
+    validations.push({
+      title: 'The node is stopping maintenance.',
+      description:
+        'The node cannot be powered off gracefully until it is either in maintenance or healthy.',
```

Comment:
```suggestion
        'The node cannot be powered off gracefully while it is exiting maintenance.',
```

This is a weird scenario, but 🤷‍♂️

Reply:
Eagle eye :-) Thank you!